File: wasm/bmxt-core/src/cmd/help_cmd.rs
//! `help` のレジストリ行と `run`。

use crate::meta::Cmd;
use crate::model::DispatchJson;
use crate::registry::api;

pub const CMD: Cmd = Cmd {
    name: "help",
    aliases: &["?"],
    usage_primary: "help",
};

pub fn run(_args: &[String]) -> DispatchJson {
    DispatchJson::lines(api::build_help_lines())
}

File: lib/features/bmxt-core/cmd/help_cmd.ts
import { getRunLocale } from "../../setting/i18n/run-locale"
import type { CmdMeta } from "../types"
import { linesDispatch } from "../types"
import { buildHelpLines } from "../registry"

export const CMD: CmdMeta = {
  name: "help",
  aliases: ["?"],
  usagePrimary: "help"
}

export function run(_args: string[]) {
  return linesDispatch(buildHelpLines(getRunLocale()))
}
